nixos/modules/services/cluster/k3s/default.nix

```diff
@@ -87,6 +88,10 @@ in
         Delegate = "yes";
         Restart = "always";
         RestartSec = "5s";
+        LimitNOFILE = 1048576;
```

Comment:
For context, these were removed based on a review on the original PR here: https://github.com/NixOS/nixpkgs/pull/84783#discussion_r409353255

I'm still mildly in favor of having them here, but @nlewo might feel more strongly that they shouldn't be here by default.

Reply:
@nlewo any thoughts? I don't feel strongly about this either, so I'm fine with reverting that part.

Reply:
Looks good to me.